src/workerd/jsg/jsg.h

```diff
@@ -2354,13 +2354,14 @@ class Lock {
       } catch (JsExceptionThrown&) {
         // If tryCatch.HasCaught() is false, it typically means that JsExceptionThrown
         // was thrown without an exception actually being scheduled on the isolate.
-        // This may happen in particular when the JsExceptionThrows was the result of
-        // TerminateException() but V8 has since cleared the terminate flag because all
+        // This may happen in particular when the JsExceptionThrown was the result of
+        // TerminateExecution() but V8 has since cleared the terminate flag because all
```

Comment:
@jasnell what exactly is this comment referring to? Is it referencing [this V8 logic here](https://github.com/v8/v8/blob/main/src/api/api.cc#L2956)? I couldn't easily figure out under exactly what circumstances `CallDepthIsZero()` would be true since comments seemed to suggest that it pertains to "V8 API" call depth rather than JS call depth.